.circleci/config.yml

```diff
@@ -134,13 +134,12 @@ jobs:
       # Code coverage upload using Codecov
       # See options explanation here: https://docs.codecov.com/docs/codecov-uploader
       - codecov/upload:
-          file: ci/unit-test/build/reports/jacoco/platformUnitTestJacocoReport/platformUnitTestJacocoReport.xml
+          file: code/edge/build/reports/jacoco/platformUnitTestJacocoReport/platformUnitTestJacocoReport.xml
```

Comment:
How about use
code/edge/build/reports/coverage/test/phone/debug/report.xml

Reply:
That directory doesn't exist in Edge when running the unit tests. Keep in mind I'm only renaming the Makefile targets and I haven't changed the Gradle files. If the common Gradle files change how Jacoco is setup, then these file locations may change later.

Reply:
I've updated the Gradle task used to generate the coverage reports and the generated report location. The report location now matches what you suggested. Thanks!